### PR TITLE
Edits to the New Application section

### PR DESCRIPTION
**Pull request by bfallonf**
@tpoitras FYI

### CODE REVIEW

**Comment by tpoitras** [comment]
@bfallonf can you possibly file.bne a render of this for me, please?

File: dev_guide/new_app.adoc [hunk 1]
```diff
@@ -12,100 +12,82 @@ toc::[]
...
-provide load balanced access to the deployment that is running your image.
-
-[NOTE]
-====
-If you xref:specifying-source-code[specify source code], you may need to
-xref:builds.adoc#starting-a-build[run a build] with `oc start-build` after the
-application is created.
-====
+You can create a new {product-title} application by using the CLI or the web
+console. Applications can be created from source code, images, or templates.
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: Recommend reversing the sentences and making "CLI" and "Web console" bullet points after a colon, like this:

{product-title} applications can be created from source code, images, or templates. You can create a new application by using the:
- CLI (xref to [[using-the-cli]])
- Web console (xref to [[using-the-web-console-na]])

File: dev_guide/new_app.adoc [hunk 2]
```diff
@@ -12,100 +12,82 @@ toc::[]
...
+configuration] to deploy the new image, and a
+xref:../architecture/core_concepts/pods_and_services.adoc#services[service] to
+provide load-balanced access to the deployment running your image.
+
+{product-title} automatically xref:build-strategy-detection[detects] whether the
+`*Docker*` or `*Source*`
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#builds[build
+strategy] is being used, and in the case of `*Source*` builds,
+xref:language-detection[an appropriate language builder image].
+
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: ...appropriate language builder image is detected.

?

File: dev_guide/new_app.adoc [hunk 3]
```diff
@@ -12,100 +12,82 @@ toc::[]
...
-If `new-app` finds a *_Dockerfile_* in the repository, it generates a
+If a *_Dockerfile_* is in the repository when creating a new application,
+{product-title} generates a
 xref:../architecture/core_concepts/builds_and_image_streams.adoc#docker-build[`*Docker*`
 build strategy]. Otherwise, it generates a
 xref:../architecture/core_concepts/builds_and_image_streams.adoc#source-build[`*Source*`
-strategy]. To use a specific strategy, set the `--strategy` flag to either
+strategy].
+
+You can use a specific strategy by setting the `--strategy` flag to either
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: s/You can use a specific strategy/You can specify the strategy

File: dev_guide/new_app.adoc [hunk 4]
```diff
@@ -171,11 +155,10 @@ $ oc new-app openshift/ruby-20-centos7:latest~/home/user/code/my-ruby-app
...
 
-=== Specifying an Image
+=== Creating an Application From an Image
 
-The `new-app` command generates the necessary artifacts to deploy an existing
-image as an application. Images can come from image streams in the
-{product-title} server, images in a specific registry or
+You can deploy an application from an existing image. Images can come from image
+streams in the {product-title} server, images in a specific registry or
 https://registry.hub.docker.com[Docker Hub], or images in the local Docker
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: is the double "or" thing here weird, or is it just me?

File: dev_guide/new_app.adoc [hunk 5]
```diff
@@ -300,39 +327,31 @@ $ oc new-app openshift/postgresql-92-centos7 \
...
-=== Specifying Labels
+==== Specifying Labels
 
 When generating applications from xref:specifying-source-code[source],
-xref:specifying-an-image[images], or xref:specifying-a-template[templates],
-you can use the `l|--label` argument to add labels to objects created by
-`new-app`. This is recommended, as labels make it easy to collectively select,
-manipulate, and delete objects associated with the application.
+xref:specifying-an-image[images], or xref:specifying-a-template[templates], you
+can use the `l|--label` argument to add labels to the created objects. Labels
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: s/l|--label/-l|--label

(missing a hyphen in front of l)

File: dev_guide/new_app.adoc [hunk 6]
```diff
@@ -357,73 +375,29 @@ $ oc new-app https://github.com/openshift/ruby-hello-world --name=myapp
 
 [[object-project-or-namespace]]
 
-==== Object Project or Namespace
+==== Creating Objects in a Different Project
+
 Normally `new-app` creates objects in the current project. However, you can tell
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: s/Normally/Normally,

Comment (tpoitras) [inline dev_guide/new_app.adoc]: "tell it" seems a bit informal

File: dev_guide/new_app.adoc [hunk 7]
```diff
@@ -490,7 +460,7 @@ image::console_select_image_or_template.png["Select Builder Image"]
 ====
 Only
 xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image
-stream tags] which have the *builder* tag listed in their annotations will
+stream tags] that have the *builder* tag listed in their annotations will
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: s/will appear/appears

File: dev_guide/new_app.adoc [hunk 8]
```diff
@@ -517,7 +487,7 @@ spec:
 web console as a builder.
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: s/will appear/appears

**Comment by tpoitras** [comment]
@bfallonf I'm done making comments now

File: dev_guide/new_app.adoc (hunk 1 above)
Comment (bfallonf) [inline dev_guide/new_app.adoc]: Agree with sentence arranging, not so much with bullet point. Done.

File: dev_guide/new_app.adoc (hunk 4 above)
Comment (bfallonf) [inline dev_guide/new_app.adoc]: It's not _wrong_, but i added "Docker Hub registry" to make it make more sense.

File: dev_guide/new_app.adoc [hunk 9]
```diff
@@ -12,100 +12,82 @@ toc::[]
...
-provide load balanced access to the deployment that is running your image.
-
-[NOTE]
-====
-If you xref:specifying-source-code[specify source code], you may need to
-xref:builds.adoc#starting-a-build[run a build] with `oc start-build` after the
-application is created.
-====
+You can create a new {product-title} application from source code, images, or templates by using the CLI or the web
+console.
```
Comment (adellape) [inline dev_guide/new_app.adoc]: @bfallonf I noticed you remove the links for "images" and "templates". Was that because you thought it was link overkill?

Comment (bfallonf) [inline dev_guide/new_app.adoc]: @adellape basically, yeah.

There's a bit of a story to do with this PR. I'll bring it up in the meeting tonight (tomorrow, your time).

**Comment by bfallonf** [comment]
@adellape I'm thinking because there's no new information here to test out that there's no point in trying to get some sort of review (with peer review already done).

I'll agree that there are benefits in a QE of sorts to go through the docs, but I don't think this is the best time to try for it. Plus, as I said in the meeting last night, if any information changes, this will effect the idea of these edits...

I'll go forward with this as-is.

**Comment by tnguyen-rh** [comment]
@bfallonf
#2219 touches the same file but (currently) has a different set of labels.  This will likely result in merge conflicts.  Do you think we should harmonize them somehow?  (I'm thinking, use the 2219 labels here, directly, but that's because i don't understand why a 3.3 label.)

@vikram-redhat 
I think these (and the 2219) changes are applicable to online as well.  WDYT?

**Comment by adellape** [comment]
Yea, I would think this should be for the same label set as https://github.com/openshift/openshift-docs/pull/2219 unless there's some reason to hold it for 3.3.

**Comment by adellape** [comment]
+ Online, as @tnguyen-rh mentions.

**Comment by tnguyen-rh** [comment]
@adellape @bfallonf @vikram-redhat 
I'll add online to 2219 and then set this PR's labels identically (removing 3.3), presuming nobody minds.  We still have a few days to adjust if need be.

**Comment by bfallonf** [comment]
Sounds like a good plan.

**Pull request merged**